tests/cluster/tests/00-base.tcl

```diff
@@ -62,3 +62,9 @@ test "Sanity for CLUSTER COUNTKEYSINSLOT" {
 test "It is possible to write and read from the cluster" {
     cluster_write_test 0
 }
+
+test "Function no-cluster flag" {
+    R 1 function load lua test {redis.register_function('f1', function() return 'hello' end, {'no-cluster'})}
```

Comment:
https://github.com/redis/redis/runs/4823518624?check_suite_focus=true#step:9:36
```
00:32:26> Function no-cluster flag: ERR Error registering functions: @user_function: 1: wrong number of arguments to redis.register_function
    while executing
"[Rn $n] {*}$args"
    (procedure "R" line 2)
    invoked from within
"R 1 function load lua test {redis.register_function('f1', function() return 'hello' end, {'no-cluster'})}"
```

I fix it in #10115